mbed_greentea/mbed_test_api.py

```diff
@@ -273,6 +276,8 @@ def get_binary_host_tests_dir(binary_path, level=2):
 
     if num_sync_packtes:
         cmd += ["--sync",str(num_sync_packtes)]
+    if tags:
+        cmd += ["--tag-filters", str(tags)]
```

Comment:
Since `tags` here comes from the command line untouched, Could you remove the `str(` and `)` to indicate that it's already in the correct format?